gateway/gateway_test.go

```diff
@@ -398,7 +400,6 @@ var _ = Describe("Gateway", func() {
 		conf.Set("Gateway.enableEventSchemasFeature", false)
 		c = &testContext{}
 		c.Setup()
-		c.initializeAppFeatures()
```

Comment:
Any reason we removed it from here and initializing in couple of places

Reply:
It was  initializeAppFeatures was initialising some mocks which were used in every test but I wanted to use some other mocks in my test so I had to remove this and add at other places.